src/App.scss

```diff
@@ -1 +1,29 @@
-// not empty
+@import "./variabls.scss";
+
+* {
+  margin: 0;
+  font-family: Mont, Arial, Helvetica, sans-serif;

```

Comment:
it's allowed to use only `box-sizing: border-box;` to `*` selector

Reply:
+